distribution/docs/src/main/resources/content/_metadataAttributes/extended-attributes-table.adoc

```diff
@@ -7,7 +7,7 @@
 
 == {title}
 
-.Extended Attributes: Attributes in this group relate to intelligence, search, and reconnaissance.
+.Extended Attributes: Attributes in this group relate to intelligence, search, and reconnaissance.  *Injected by default.*
```

Comment:
❓ Would it be useful mention they are injected into all `MetacardType` or link to the `29.6. Developing Attribute Injections` section?

Reply:
There are links to that section in the CDM documentation but not sure we want links here in the tables section